packages/compass/src/main/menu.spec.ts

```diff
@@ -127,6 +127,24 @@ describe('CompassMenu', function () {
           {
             type: 'separator',
           },
+          {
+            checked: true,
+            label: 'Use OS Theme (Preview)',
```

Comment:
I'm thinking let's move these options into the Compass menu -> preferences -> color theme. Instead of where they are currently in view. Will update